src/puppetlabs/i18n/core.clj

```diff
@@ -215,11 +217,13 @@
   [namespace loc msgid msgid-plural count]
   (let [bundle (get-bundle namespace loc)]
     (if bundle
```

Comment:
Now that there are other ifs nested inside this one, they should both be combined into a `cond`, because it gets to be too hard to read. This also applies above on line 203